fairscale/nn/pipe/multiprocess_pipe.py

```diff
@@ -463,7 +461,6 @@ def __init__(
 
         self.group = group
         self.worker_map = worker_map
-        self.input_device = input_device
```

Comment:
We are using this in ampnet
https://github.com/facebookresearch/fairscale/blob/master/experimental/nn/ampnet_pipe/pipe.py#L60

Reply:
Ah. Good catch. Fixed.